sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -433,18 +436,19 @@ public virtual Response<KeyVaultCertificateWithPolicy> RecoverDeletedCertificate
         /// </summary>
         /// <param name="certificateName">The name of the <see cref="DeletedCertificate"/></param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>The recovered certificate and policy</returns>
-        public virtual async Task<Response<KeyVaultCertificateWithPolicy>> RecoverDeletedCertificateAsync(string certificateName, CancellationToken cancellationToken = default)
+        /// <returns>A <see cref="RecoverDeletedCertificateOperation"/> to wait on this long-running operation.</returns>
```

Comment:
Can we add add a comment to these, which states that you don't have to wait? not sure how to word it.

Reply:
Sure. In hindsight, you had me do that for Keys and Secrets so I can just use that text.